### PR TITLE
Autogenerated 5.6.0 changes

### PR DESCRIPTION
**Pull request by joshmoore**
Likely last PR before tagging. cc: @jburel

### CODE REVIEW

**Comment by jburel** [comment]
this was run before https://github.com/ome/omeroweb-install/pull/56 was merged.
It will need to be run again

File: omero/users/history.rst [hunk 1]
```diff
@@ -32,46 +32,19 @@ omero clients:
...
-5.6.0-m1 (November 2019)
-------------------------
-
-First milestone of OMERO with support for Python 3. This version
-depends on omero-py and omero-web 5.6.0.dev3 and is intended for
-internal testing. Subsequent versions will include documentation
-of the necessary migration steps and a list of supported platforms.
-
+- omero-py 5.6.0
+- omero-web 5.6.0
```
Comment (jburel) [inline omero/users/history.rst]: 5.61

File: omero/users/history.rst [hunk 2]
```diff
@@ -32,46 +32,19 @@ omero clients:
...
+- omero-dropbox 5.6.0
```
Comment (manics) [inline omero/users/history.rst]:
```suggestion
- omero-dropbox 5.6.1
```

File: omero/users/history.rst (hunk 1 above)
Comment (manics) [inline omero/users/history.rst]:
```suggestion
- omero-web 5.6.1
```

File: omero/users/history.rst (hunk 2 above)
Comment (jburel) [inline omero/users/history.rst]: 5.6.1
This will have to be updated in ome/openmicroscopy repo

Comment (joshmoore) [reply omero/users/history.rst]: Can you go ahead and open a PR? I'll change in the docs, but I'd prefer to _not_ retag ome/openmicroscopy just to bump the history.

Comment (jburel) [reply omero/users/history.rst]: done https://github.com/ome/openmicroscopy/pull/6203

Comment (jburel) [reply omero/users/history.rst]: no need to retag

**Comment by jburel** [comment]
major problem is the omeroweb-install miss. This could be done manually here too on all the guides

**Comment by joshmoore** [comment]
Running now.

**Comment by joshmoore** [comment]
Updated.

**Review by mtbc — APPROVED** [review]

**Pull request merged**